docs/api-server-walkthrough.md

```diff
@@ -32,6 +32,16 @@ $ etcd
 $ ./bin/apiserver -v 10 --etcd-servers http://localhost:2379
 ```
 
+Alternatively, you can run the apiserver and etcd as a pod:
+```
+# run the apiserver & etcd as a pod
+$ kubectl create -f contrib/examples/apiserver/apiserver.yaml
+# enable port forwarding from localhost:6443 to the apiserver
+$ kubectl port-forward service-catalog-apiserver 6443:6443
```

Comment:
this might be a pre-req for `hack/local-up-cluster.sh`, but you'll need `socat` and `nsenter` which were not installed for me. 

still trying to find nsenter.